application/modules/ucp/views/ucp.tpl

```diff
@@ -1,7 +1,7 @@
 <section id="ucp_top">
 	<a href="{$url}ucp/avatar" id="ucp_avatar">
 		<div>{lang("change_avatar", "ucp")}</div>
-		<img src="{$avatar}" height="100" width="100"/>
+		<img src="{$avatar}" height="120" width="120"/>
```

Comment:
What if I see this in my phone? Is it responsive?

Reply:
It will be okay. I tested this on the phone too.

Reply:
The Avatar block has the size of 120x120 but the image is displayed only in 100x100. So this will not affect the block size only of the image.